### ISSUE TITLE
Wrong Translation in the German language file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#231]
```
Somebody translated "Mead" with the German word for meat = "Fleisch"
Which leads to a bottle of meat in the German version ;)

The right translation is fpr "Mead" is "Met" or "Honigwein".

BTW nice Game

```

Original issue reported on code.google.com by `webmai...@gmail.com` on 5 Aug 2011 at 1:12

**Comment by GoogleCodeExporter** [comment]
```
Thanks for your report.
This has already been reported and corrected.
```

Original comment by `SamuelPl...@gmail.com` on 5 Aug 2011 at 4:27

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 30 Aug 2011 at 9:05
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.10**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
There is still one more occurrence of the bad translation.
In line 166 of file res/values-de/content_itemlist.xml it has to be "Met mit 
Schlafmittel" instead of "Fleisch mit Schlafmittel". I would suggest calling it 
simply "Schlafmet" though.
And there is one additional typo I came across, right in the first message on 
screen. In line 206 of file res/values-de/strings.xml has to be "Vielen Dank, 
dass du..." instead of "Viele Dank, dass du..."

Thanks for the great game.

```

Original comment by `s.jano...@gmail.com` on 11 Sep 2011 at 5:43

**Comment by GoogleCodeExporter** [comment]
```
re comment 3: Thanks for the report. Fixed in r156.
```

Original comment by `oskar.wi...@gmail.com` on 12 Sep 2011 at 3:23

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
